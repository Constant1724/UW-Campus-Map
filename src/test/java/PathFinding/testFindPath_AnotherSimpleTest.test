# START_TEST
# NAME AnotherSimpleTest
# test a graph can be loaded and correctly find the path from A to B, with least
# cost, as cost being the alphabetical ordering of edges and labels..

# Create a graph
CreateGraph graph1

# AddNode
AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
AddNode graph1 D

AddEdge graph1 A B 10.000

AddEdge graph1 A C 1.000

AddEdge graph1 C D 1.000

AddEdge graph1 D B 1.000

FindPath graph1 A B
# END_TEST