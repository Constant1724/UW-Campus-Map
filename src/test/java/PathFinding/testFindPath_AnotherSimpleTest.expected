# START_TEST
# NAME AnotherSimpleTest
# test a graph can be loaded and correctly find the path from A to B, with least
# cost, as cost being the alphabetical ordering of edges and labels..

# Create a graph
created graph graph1

# AddNode
added node A to graph1
added node B to graph1
added node C to graph1
added node D to graph1

added edge 10.000 from A to B in graph1

added edge 1.000 from A to C in graph1

added edge 1.000 from C to D in graph1

added edge 1.000 from D to B in graph1

path from A to B:
A to C with weight 1.000
C to D with weight 1.000
D to B with weight 1.000
total cost: 3.000
# END_TEST